specifyweb/frontend/js_src/lib/components/FormParse/index.ts

```diff
@@ -85,6 +86,10 @@ export const getViewSetApiUrl = (viewName: string): string =>
         : undefined,
   });
 
+export const clearViewLocal = (viewName: string) => {
+  views = removeKey(views, viewName);
```

Comment:
(optional) `views[viewName] = undefined;` - then you don't have to change `views` from `let` to `const`